docs/analytics/object_stores/local.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Local File System
+---
+
+## Overview
+
+This code block demonstrates how to create a foreign table over the local file system. Note that this only works
```

Comment:
Do you think it's worth mentioning here that this is the "equivalent" of what `pg_analytics` did before? Perhaps it can help our users of `pg_analytics` translate more easily to `pg_lakehouse`?

Reply:
I stayed away from that kind of strong language bc it's not quite the equivalent